server/src/server.ts

```diff
@@ -121,9 +121,9 @@ parser.setTableFetch(async (table: string, aliases = false) => {
 
 		currentSubfield.keywords = [getPrettyType({
 			type,
-			len: digits === 0 ? strLength : digits,
+			len: digits === `0` ? strLength : digits,
```

Comment:
I am not sure this is right.

`digits` is always a number and a strict comparison (`===`) against `'0'` will always return false.

Reply:
<img width="707" alt="image" src="https://user-images.githubusercontent.com/3708366/198667499-8929d090-b7d0-446b-9cfb-1c562228de2f.png">

Reply:
Another example digits is a number. I can confirm `digits` is always a number and so this change could be wrong. 
<img width="707" alt="image" src="https://user-images.githubusercontent.com/3708366/198668597-57750bf6-440c-440a-a1d6-271052b07f8f.png">